### ISSUE TITLE
 Source param shape is 96 3 5 5 (7200); target param shape is 96 1 5 5 (2400). 

### ISSUE THREAD

**Issue by iscas-lee** [Tonyfy/LCNN_TRAIN#12]
When i used the trained model. it appears

F0914 14:49:38.433799  1816 net.cpp:773] Cannot copy param 0 weights from layer 'conv1'; shape mismatch.  Source param shape is 96 3 5 5 (7200); target param shape is 96 1 5 5 (2400). To learn this layer's parameters from scratch rather than copying from a saved net, rename the layer.
**\* Check failure stack trace: ***

**Comment by Tonyfy** [comment]
we use gray image

**Comment by iscas-lee** [comment]
Thank you very much.  This problem has been solved.
However , another problem is  that my training process is  not convergence. the **train accuracy** is always between 0.80 ~ 0.90, I  have no idea what is wrong .

I0917 21:27:52.891999 15704 solver.cpp:337] Iteration 2095000, Testing net (#0)
I0917 21:27:53.789000 15704 solver.cpp:391] Test loss: 1.09674
I0917 21:27:53.789000 15704 solver.cpp:404]     Test net output #0: accuracy = 0.84
I0917 21:27:53.835999 15704 solver.cpp:228] Iteration 2095000, loss = 0.816846
I0917 21:27:53.835999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.765625
I0917 21:27:53.835999 15704 sgd_solver.cpp:106] Iteration 2095000, lr = 0.000160393
I0917 21:28:06.017999 15704 solver.cpp:228] Iteration 2095100, loss = 0.850126
I0917 21:28:06.017999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.796875
I0917 21:28:06.017999 15704 sgd_solver.cpp:106] Iteration 2095100, lr = 0.000160388
I0917 21:28:18.228999 15704 solver.cpp:228] Iteration 2095200, loss = 0.712411
I0917 21:28:18.228999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.84375
I0917 21:28:18.228999 15704 sgd_solver.cpp:106] Iteration 2095200, lr = 0.000160383
I0917 21:28:30.428999 15704 solver.cpp:228] Iteration 2095300, loss = 0.949643
I0917 21:28:30.428999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.75
I0917 21:28:30.428999 15704 sgd_solver.cpp:106] Iteration 2095300, lr = 0.000160377
I0917 21:28:42.927000 15704 solver.cpp:228] Iteration 2095400, loss = 0.838903
I0917 21:28:42.927000 15704 solver.cpp:244]     Train net output #0: accuracy = 0.78125
I0917 21:28:42.927000 15704 sgd_solver.cpp:106] Iteration 2095400, lr = 0.000160372
I0917 21:28:55.188999 15704 solver.cpp:228] Iteration 2095500, loss = 0.592464
I0917 21:28:55.188999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.859375
I0917 21:28:55.188999 15704 sgd_solver.cpp:106] Iteration 2095500, lr = 0.000160367
I0917 21:29:07.310999 15704 solver.cpp:228] Iteration 2095600, loss = 0.608305
I0917 21:29:07.310999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.84375
I0917 21:29:07.310999 15704 sgd_solver.cpp:106] Iteration 2095600, lr = 0.000160362
I0917 21:29:19.417999 15704 solver.cpp:228] Iteration 2095700, loss = 0.581019
I0917 21:29:19.417999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.828125
I0917 21:29:19.417999 15704 sgd_solver.cpp:106] Iteration 2095700, lr = 0.000160356
I0917 21:29:31.516999 15704 solver.cpp:228] Iteration 2095800, loss = 0.717885
I0917 21:29:31.516999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.859375
I0917 21:29:31.516999 15704 sgd_solver.cpp:106] Iteration 2095800, lr = 0.000160351
I0917 21:29:43.632999 15704 solver.cpp:228] Iteration 2095900, loss = 1.12747
I0917 21:29:43.632999 15704 solver.cpp:244]     Train net output #0: accuracy = 0.765625
I0917 21:29:43.632999 15704 sgd_solver.cpp:106] Iteration 2095900, lr = 0.000160346
I0917 21:29:55.631999 15704 solver.cpp:337] Iteration 2096000, Testing net (#0)

**Comment by Tonyfy** [comment]
if your use the same parameters ,and get this stable accurate of 0.8~0.9(good but little smaller than mine),which may cause by bad-image-preprocessing.
